akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -340,7 +340,8 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
   val `application/grpc+proto`                                                    = abin("grpc+proto", NotCompressible)
   val `application/lha`                                                           = abin("lha", NotCompressible, "lha")
   val `application/lzx`                                                           = abin("lzx", NotCompressible, "lzx")
-  val `application/mspowerpoint`                                                  = abin("mspowerpoint", NotCompressible, "pot", "pps", "ppt", "ppz")
+  val `application/mspowerpoint`                                                  = abin("mspowerpoint", NotCompressible, "pot", "pps", "ppz")
+  val `application/vnd.ms-powerpoint`                                             = abin("vnd.ms-powerpoint", NotCompressible, "ppt")
```

Comment:
According to this [official table](https://blogs.msdn.microsoft.com/vsofficedeveloper/2008/05/08/office-2007-file-format-mime-types-for-http-content-streaming-2/), the other extensions should be moved as well.

```suggestion
  val `application/vnd.ms-powerpoint`                                             = abin("vnd.ms-powerpoint", NotCompressible, "pot", "pps", "ppt", "ppz")
```

Reply:
Also, we keep the list alphabetical, so could you move it down to its right place?